src/fetch/transfer_file.py

```diff
@@ -223,6 +216,7 @@ def _process_native_transfers(
             if slippage is not None:
                 try:
                     transfer.add_slippage(slippage)
+                    # TODO - this should also be in WEI.
```

Comment:
Yeah, probably you are right...

Reply:
Yes, Although I think I would prefer to do this in a follow up.

Reply:
Made an issue:

https://github.com/cowprotocol/solver-rewards/issues/76